tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "es2021",
-    "module": "commonjs",
+    "module": "Node16",
```

Comment:
are you sure this is what we want need? why? cf [docs](https://www.typescriptlang.org/tsconfig#node16nodenext-nightly-builds)

Reply:
[q] Ah, is it the same? According to documentation it should be, but I thought I saw it was different. That's great then 😊